### PR TITLE
reddi2019on_2+joshpapermaster

### PR DESCRIPTION
**Pull request by joshpapermaster**

### CODE REVIEW

File: _summaries/reddi2019on_2.md [hunk 1]
```diff
@@ -1,19 +1,61 @@
...
+
+There are three theorems presented in this paper, with each building upon the last, that provide the structure behind the issues with ADAM not converging:
+
+1.
+There is an online convex optimization problem where ADAM has non-zero average regret 
+- The details are provided by the situations provided in the following theorems
+- Non-zero average regret basically means the influence of the gradient can't keep up with bad decisions, thus the the network is not converging or learning anything
+
+2.
+For any constant β1, β2 ∈ [0, 1) such that β1 < β2^(1/2), there is an online convex optimization problem where ADAM has non-zero average regret
```
Comment (timchen0618) [inline _summaries/reddi2019on_2.md:28]: probably should talk about what β1, β2 are

File: _summaries/reddi2019on_2.md [hunk 2]
```diff
@@ -1,19 +1,61 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{reddi2019on}}
```
Comment (timchen0618) [inline _summaries/reddi2019on_2.md:4]: overall I think the review is very good. I think you cover every important point in the paper, and you mostly use your own words to illustrate the ideas. Just some minor stuff needed to be clarified. Also theorem part is too lengthy for me.

Comment (joshpapermaster) [reply _summaries/reddi2019on_2.md:4]: @timchen0618 Thanks, I updated my summary following your comments. Let me know if there's anything else you think I should add

File: _summaries/reddi2019on_2.md [hunk 3]
```diff
@@ -1,19 +1,61 @@
...
+
+The paper also presents an alternative solution called ADAMNC
+- The main idea is to dynamically change β1 and β2 so that the conditions for non-convergence are never met for long
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+- There are certain situations that come up practically where ADAM slows down or entirely fails to converge
```
[the remark this answers is not in the document]
Comment (timchen0618) [reply _summaries/reddi2019on_2.md:54]: Here I'm actually thinking that you should add why ADAM fails (the exp moving average) in the first bullet point. I think the 2nd bullet point is fine.

File: _summaries/reddi2019on_2.md (hunk 2 above)
Comment (timchen0618) [reply _summaries/reddi2019on_2.md:4]: I think it is great overall! Only one thing about TL;DR, I added a comment to that.
